bin-src/git/findAncestorBuildWithCommit.ts

```diff
@@ -55,15 +55,17 @@ export async function findAncestorBuildWithCommit(
   let skip = 0;
   while (skip < limit) {
     // eslint-disable-next-line no-await-in-loop
-    const { app } = await client.runQuery(AncestorBuildsQuery, {
+    const { app } = (await client.runQuery(AncestorBuildsQuery, {
       buildNumber,
       skip,
       limit: Math.min(page, limit - skip),
-    });
+    })) as AncestorBuildsQueryResult;
```

Comment:
This was already fixed [here](https://github.com/chromaui/chromatic-cli/commit/b7a250f8a292befee3441cce5d0514ecf7e115a6).